packages/bpk-tokens/src/base/typography.json

```diff
@@ -262,6 +262,126 @@
       "value": "{!WHITE}",
       "type": "color",
       "category": "text-colors"
+    },
+    "TEXT_XS_FONT_SIZE": {
+      "value": "{!FONT_SIZE_SM}",
+      "type": "size",
+      "category": "font-sizes"
+    },
+    "TEXT_XS_LINE_HEIGHT": {
+      "value": "{!SPACING_MD}",
+      "type": "number",
+      "category": "line-heights"
+    },
+    "TEXT_XS_FONT_WEIGHT": {
+      "value": "400",
+      "type": "number",
+      "category": "font-weights"
+    },
+    "TEXT_XS_LETTER_SPACING": {
+      "value": "normal",
+      "type": "size",
+      "category": "letter-spacings"
+    },
+    "TEXT_SM_FONT_SIZE": {
+      "value": "{!FONT_SIZE_SM}",
+      "type": "size",
+      "category": "font-sizes"
+    },
+    "TEXT_SM_LINE_HEIGHT": {
+      "value": "{!SPACING_MD}",
+      "type": "number",
+      "category": "line-heights"
+    },
+    "TEXT_SM_FONT_WEIGHT": {
+      "value": "400",
+      "type": "number",
+      "category": "font-weights"
+    },
+    "TEXT_SM_LETTER_SPACING": {
+      "value": "normal",
```

Comment:
Yeah it's definitely confusing and the more I think of it it feels like we're trying to shoehorn two distinct dimensions into one. What we really have is theme and platform which are distinct dimensions and maybe having a single value for every token in a theme isn't enough. 

Using spacing as an example

| Platform                  | base         | iOS  |
| ------------------------- | ------------ | ---- |
| Web(iOS/Android webviews) | .375rem(6px) | 4px  |
| iOS(Native and RN)        | -            | 4pt  |
| Android(Native and RN)    | -            | -    |



Is platform specific formatters enough to transform a platform agnostic value into a platform specific one? In the above example `SPACING_XS` would be a `4pt` for `iOS` and we'd use a `1px = 1pt` transformation to convert to pixels for web and then possibly another calculation to get a rem value which works. But like you mention to we need platform specific tokens for values such as kerning/letter spacing because the semantics of the different platforms are different enough to make it impossible to transform an agnostic value into a specific one?